src/components/yourLikesList/YouLikesProfiles.tsx

```diff
@@ -2,40 +2,57 @@ import { Grid, Typography } from '@mui/material'
 import { getColumns } from '../../helpers/helper'
 import UserListRenderer from './YourLikesUserListRenderer'
 import useUsersData from 'hooks/useUsersData'
-import { nearByWhoLikedMeStyles } from '../../styles/nearByWhoLikedMeStyles'
+import { makeStyles } from 'tss-react/mui'
+import theme from '../../styles/createTheme'
+
 type IsMobileProps = {
   isMobile: boolean
 }
 
 const YouLikesProfiles = ({ isMobile }: IsMobileProps) => {
-  const { classes } = nearByWhoLikedMeStyles()
   const columns = getColumns(isMobile)
-  const txtAlign = isMobile ? 'center' : 'left'
-
   const { data: profilesList } = useUsersData()
+  const { classes } = useStyles()
 
   return (
     <Grid container spacing={3}>
       <Grid item xs={12}>
-        <Typography className={classes.headerNear} sx={{ textAlign: txtAlign }}>
+        <Typography variant="h1" className={classes.title}>
           Your likes list
         </Typography>
-        <Typography
-          className={classes.description}
-          sx={{ textAlign: txtAlign }}
-        >
+        <Typography variant="body2" className={classes.description}>
           These people have already liked you – just like them back and it’s a
           match!
         </Typography>
       </Grid>
       <Grid item xs={12}>
-        <UserListRenderer
-          users={profilesList}
-          classes={classes}
-          columns={columns}
-        />
+        <UserListRenderer users={profilesList} columns={columns} />
       </Grid>
     </Grid>
   )
 }
 export default YouLikesProfiles
+
+const useStyles = makeStyles()(() => {
+  return {
+    title: {
+      paddingTop: '60px',
+      paddingBottom: '10px',
+      textAlign: 'center',
+      [theme.breakpoints.up('md')]: {
+        textAlign: 'left',
+      },
+    },
+    description: {
+      textAlign: 'center',
+      [theme.breakpoints.up('md')]: {
+        textAlign: 'left',
+      },
+      [theme.breakpoints.down('sm')]: {
```

Comment:
Is this breakpoint necessary? It seems, that without this breakpoint text looks good.

Reply:
It's used not only in my screens but also on pages with no errors. It's the common/same heading and it's working as it is in design "Web version. Likes list". If the design has to be different, it should be either discussed with Olga and the web version of the error pages should be provided in design.